src/Apps/Artwork/Components/PrivateArtwork/PrivateArtworkMetadata.tsx

```diff
@@ -0,0 +1,107 @@
+import { graphql, useFragment } from "react-relay"
+import { PrivateArtworkMetadata_artwork$key } from "__generated__/PrivateArtworkMetadata_artwork.graphql"
+import {
+  Box,
+  Column,
+  GridColumns,
+  HTML,
+  ReadMore,
+  Spacer,
+  Text,
+} from "@artsy/palette"
+
+interface PrivateArtworkMetadataProps {
+  artwork: PrivateArtworkMetadata_artwork$key
+}
+
+export const PrivateArtworkMetadata: React.FC<PrivateArtworkMetadataProps> = ({
+  artwork,
+}) => {
+  const data = useFragment(
+    graphql`
+      fragment PrivateArtworkMetadata_artwork on Artwork {
+        title
+      }
+    `,
+    artwork
+  )
```

Comment:
when you say that this page is static, you're saying that you're just setting up the bare bones layout, but this fragment will be updated to include the actual data that we need for each of these components, correct @damassi ?

Reply:
Not a strict rule of course, but this really helps speed things along since one can focus on exactly the stage one is on, without mixing all the concerns all together. Instead of architecture, data, design / css, etc, each pr is one thing and one thing only.

Reply:
Not a strict rule of course, but this really helps speed things along since one can focus on exactly the stage one is on, without mixing all the concerns all together. Instead of architecture, data, design / css, etc, each pr is one thing and one thing only.